src/configure.ac

```diff
@@ -2430,6 +2430,16 @@ then
     AC_MSG_ERROR([fuser not found])
 fi
 
+AC_PATH_PROG(YAPPS2, yapps2, "none")
+if test $YAPPS2 = "none"; then
+    AC_PATH_PROG(YAPPS, yapps, "none")
+    if test $YAPPS = "none"; then
+        AC_MSG_ERROR([No yapps2 or yapps executable found])
+    else
+        YAPPS2=$YAPPS
+    fi
+fi
+
```

Comment:
@zultron 
Since Makefile.inc.in contains YAPP2=@YAPPS2@, don't see how this works without an AC_SUBST(YAPPS2)?

Reply:
https://www.gnu.org/software/autoconf/manual/autoconf-2.68/html_node/Generic-Programs.html

> Macro: AC_PATH_PROG (variable ,[...])
>  Like AC_CHECK_PROG [...]

and

> Macro: AC_CHECK_PROG (variable, [...])
> [...]  Calls AC_SUBST for variable. [...]

I didn't know this, either!  It does work; check `Makefile.inc` after running `./configure`.

Reply:
Well you live and learn.
Not exactly transparent but so long as it does get substituted and it is not working because yapps existed anyway, which was my concern.